src/Ema/Server.hs

```diff
@@ -357,7 +357,10 @@ wsClientShim =
                 // This is a new route switch; scroll up.
                 window.scrollTo({ top: 0});
                 routeVisible = document.location.pathname;
-              } 
+              }
+              if (window.location.hash) {
+                window.location.hash = window.location.hash;
+              }
```

Comment:
Huh. What does this do? @lucasvreis

Reply:
See what I said in the comment above ([here](https://github.com/srid/ema/pull/83#issuecomment-1038268688)). Sorry but I think this is not a hack, this just forces a hashchange event when a hash is present. A more explicit way is to write `window.dispatchEvent(new HashChangeEvent("hashchange"))`.

Reply:
@srid sorry, while this works in Firefox this does not work e.g. in Chrome. So I apologize and suggest you to use `window.dispatchEvent(new HashChangeEvent("hashchange"))` instead of this line (that should work in all browsers)

Reply:
@lucasvreis Don't worry, this is just my fault for merging in haste without looking at new diff (my brain had been fried at the end of a long day). I would have normally asked you to change it to the code you suggested before merging it.  

I'll test out your code later. 👍🏿 I also think we should update the demo to show support for url anchor navigation.